client/src/entrypoints/admin/userbar.js

```diff
@@ -2,45 +2,47 @@
 // Please stick to old JS APIs and avoid importing anything that might require a vendored module
 // More background can be found in webpack.config.js
 
-document.addEventListener('DOMContentLoaded', (e) => {
+document.addEventListener('DOMContentLoaded', () => {
   const userbar = document.querySelector('[data-wagtail-userbar]');
   const trigger = userbar.querySelector('[data-wagtail-userbar-trigger]');
   const list = userbar.querySelector('.wagtail-userbar-items');
-  const className = 'is-active';
-  const hasTouch = 'ontouchstart' in window;
+  const listItems = list.querySelectorAll('li');
```

Comment:
I don't think every list item has to be a `menuitem` necessarily. It could be a `menuitemradio` or a `menuitemcheckbox`.

What reason would there be for to selecting something else then the `<li>`s? If you're thinking of developers adding their custom actions to the userbar, I think it's unlikely they will override `item_base.html`.